pom.xml

```diff
@@ -236,8 +236,10 @@
     <version.com.fasterxml.jackson.core>2.5.4</version.com.fasterxml.jackson.core>
     <!-- keep in sync with modules/system/layers/base/com/google/guava of the current WildFly/EAP -->
     <version.com.google.guava>18.0</version.com.google.guava>
+    <!-- com.wordnik is deprecated. Use io.swagger instead. -->
     <version.com.wordnik.swagger>1.3.13</version.com.wordnik.swagger>
     <version.io.gatling.highcharts>2.1.6</version.io.gatling.highcharts>
+    <version.io.swagger>1.5.8</version.io.swagger>
```

Comment:
Micro upgrades should be fine. But I admit I have not tested with 1.5.8. Anyway, if there are problems with 1.5.8, one can always override `<version.io.swagger>1.5.8</version.io.swagger>` in the child repo.